dev/AppNotifications/NotificationProgressData.h

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include <frameworkUdk/ToastNotificationsRT.h>
+
+struct NotificationProgressData : winrt::implements<NotificationProgressData, ::ABI::Microsoft::Internal::ToastNotifications::IToastProgressData>
```

Comment:
Why did we use HSTRING here instead of the new types?

Reply:
@sharath2727 or @danielayala94

Reply:
Which new types? I guess the question here is "why this is not implemented in the winrt style like the SDK types?"

This is an ABI interface, declared in the FrameworkUdk. There is no projection (winmd) giving us winrt-styled functions. @sharath2727 actually looked into it and there was no straightforward solution (if any) to generate a winmd for these UDK interfaces. We decided to go this way instead.